generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-detail.component.html

```diff
@@ -68,9 +68,13 @@
             </span>
             <%_ } else { _%>
             <%_ if (dto == 'no') { _%>
-            <a [routerLink]="['<%= otherEntityStateName %>-detail', <%= entityInstance + '.' + relationshipFieldName + '?.id' %>]">{{<%= entityInstance + "." + relationshipFieldName + "?." + otherEntityField %>}}</a>
+            <div *ngIf="<%= entityInstance + '.' + relationshipFieldName %>">
```

Comment:
We don't need it in user-management as the login is required so there's no way it could be null